app/presenters/finder_content_item_presenter.rb

```diff
@@ -50,6 +50,10 @@ def details
       summary: metadata.fetch("summary", nil),
       facets: schema.fetch("facets", []),
     }
+    unless metadata["default_order"].nil?
+      details_hash[:default_order] = metadata["default_order"]
```

Comment:
It matters as much as it does anywhere else.

If the objection is to the use of `unless metadata["default_order"].nil?`, sure it's verbose and could be changed to `if metadata["default_order"]`, but it read clearly already.